docs/static/settings-file.asciidoc

```diff
@@ -227,6 +227,11 @@ Values other than `disabled` are currently considered BETA, and may produce unin
   would increase the size of the dead letter queue beyond this setting.
 | `1024mb`
 
+| `dead_letter_queue.storage_policy`
+| Defines the behavior to apply when the dead_letter_queue.max_bytes is reached. When an event would increase the size
+  over `dead_letter_queue.max_bytes` the event is dropped if valued `drop_newer` or remove the oldest segment if `drop_older`.
```

Comment:
```suggestion
| Defines the action to take when the dead_letter_queue.max_bytes setting is reached: `drop_newer` stops accepting new values that would push the file size over the limit, and `drop_older` removes the oldest events to make space for new ones.
```

Reply:
Sometimes tables get weird when github suggestions are applied. If you decide to take this suggestion, you can check out the preview with this link: https://logstash_13923.docs-preview.app.elstc.co/guide/en/logstash/master/logstash-settings-file.html. Or I will if I get another chance to look. :-)